Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
```suggestion

```

Reply:
If you leave out the version, it will allow the app to run on multiple versions.